config.json

```diff
@@ -1656,6 +1656,15 @@
         "prerequisites": [],
         "difficulty": 8,
         "topics": ["recursion", "searching", "trees"]
+      },
+      {
+        "slug": "zebra-puzzle",
+        "name": "Zebra Puzzle",
+        "uuid": "3a34cb49-546f-4aae-b49e-7d5b25db7ebb",
+        "practices": [],
+        "prerequisites": [],
```

Comment:
Please think about which concepts the student needs to know to write a reasonable solution for this exercise and fill this array. You can find a list of existing concepts here: https://github.com/exercism/javascript/blob/main/reference/roadmap.md#concept-that-exists-or-an-issue-has-been-created